sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -113,6 +123,22 @@ def __init__(self, type_uri=CUSTOM_REFERENCE_NS, identity="customReferenceProper
             self.location = sbol3.OwnedObject(self, f"{self.CUSTOM_REFERENCE_NS}#location", 0, math.inf, type_constraint=sbol3.Range)
 
 
+    class CustomStructuredCommentProperty(sbol3.CustomTopLevel):
+        """Serves to store information and annotations for 'Structured_Comment' objects in 
```

Comment:
I think it would be good for the docstring to include a pointer to NCBI's reference page on structured comments: https://www.ncbi.nlm.nih.gov/genbank/structuredcomment/

Reply:
I have put up the link in the docstring in the latest commit